src/content/landing-rtl.ts

```diff
@@ -86,6 +86,18 @@ export const getRtlLanding = (t: TFunction): CustomPageContent => ({
                 },
             ],
         },
+        {
+            type: CustomBlock.Libraries,
```

Comment:
Can we fix this?

<img width="1347" alt="image" src="https://github.com/user-attachments/assets/54403ef9-ead7-497f-9b5e-4568f40da7a5" />